### ISSUE TITLE
Error on https://armorgames.com/play/10175/arkandian-crusade?tag-referral=rpg

### ISSUE THREAD

**Issue by RomHac** [ruffle-rs/ruffle#2968]
# Error Info
Error name: Error
Error message: panicked at 'index out of bounds: the len is 0 but the index is 4294967295', core/src/avm1/object/script_object.rs:807:17
Error stack:
```
Error: panicked at 'index out of bounds: the len is 0 but the index is 4294967295', core/src/avm1/object/script_object.rs:807:17
    at s.wbg.__wbg_new_df6e6ab7a65c4c4d (<anonymous>:3:6312)
    at std::sync::once::Once::call_once::{{closure}}::h7141d078b1ea013f (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[1277]:0x2570ef)
    at std::sync::once::Once::call_inner::h02eac160d339afc2 (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[1536]:0x28688a)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hf48d3d3ee3d10801 (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[4372]:0x35d891)
    at std::panicking::rust_panic_with_hook::h123718ba3bf480af (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[3313]:0x335027)
    at std::panicking::begin_panic_handler::{{closure}}::hf393a82e58397bd2 (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[3929]:0x352215)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h2ff2cfc953878925 (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[4705]:0x362bff)
    at rust_begin_unwind (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[4584]:0x3613d1)
    at core::panicking::panic_fmt::h285dc015a87be5a4 (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[4707]:0x362c63)
    at core::panicking::panic_bounds_check::heedacfd10d9ec31d (chrome-extension://jhcpiagfnilbbhmgaimfpocclinhkbga/dist/90d03b7718fde93d787334ae45f283f3.wasm:wasm-function[4115]:0x35798f)
```

# Player Info
Allows script access: true

# Page Info
Page URL: https://armorgames.com/play/10175/arkandian-crusade?tag-referral=rpg
SWF URL: https://cache.armorgames.com/files/games/arkandian-crusade-10175.swf?v=1373587524

# Browser Info
Useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.96 Safari/537.36
OS: MacIntel

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-30
Channel: nightly
Built: 2021-01-30T00:35:10.218Z
Commit: 02dfb23c6738bc5a576113b3f1794127bc7ae84e

**Comment by Poopooracoocoo** [comment]
Should be fixed by #3099 soon! :)

**Comment by Herschel** [comment]
Fixed by #3099.
